### DISCUSSION TITLE
浅忆我的本科三年 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#22] [Announcements]
# 浅忆我的本科三年 🐱 ネコのメモ帳

本文作于 2024/09/24。

https://nova.gal/blog/2024/09/24/%E6%B5%85%E8%B0%88%E6%88%91%E7%9A%84%E6%9C%AC%E7%A7%91%E4%B8%89%E5%B9%B4

**Comment by nyyyddddn** [comment]
好厉害，羡慕

**Comment by ZephyrVictor** [comment]
好强，顺便问问vul337实习的过程中压力大不大啊 push嘛 还有后续去pku的bar高嘛QAQ
